src/javascript/binary/base/client.js

```diff
@@ -98,8 +98,9 @@ const Client = (() => {
         const is_ico_only = get('is_ico_only', loginid);
         return ((
             (type === 'virtual' && this_type === 'virtual') ||
-            (type === 'real'    && this_type !== 'virtual') ||
-            type === this_type) && !is_ico_only &&              // Account shouldn't be ICO_ONLY.
+            (type === 'real'    && this_type !== 'virtual' && !is_ico_only) ||
+            (type === 'ico'     && is_ico_only) ||
+            type === this_type) &&              // Account shouldn't be ICO_ONLY.
```

Comment:
should we remove the comment now that we want to return ico accounts as well? or move it to line 101 as it's only related to that line